packages/drivers/debugger/package.json

```diff
@@ -107,14 +107,8 @@
 		"rimraf": "^4.4.0",
 		"typescript": "~5.4.5"
 	},
-	"fluidBuild": {
-		"tasks": {
-			"tsc": [
-				"typetests:gen"
-			]
-		}
-	},
```

Comment:
These changes won't make the build invalid before the tools update comes through?
I think unfortunately these are a post-update or with-update changes. BTW is the policy updated with the typetest changes coming?

Reply:
These are safe to make separate from the 0.45 upgrade. There are corresponding changes to the fluid build task config.